src/copyright/ui/HistogramBase.php

```diff
@@ -234,7 +234,7 @@ public function Output()
     $dataset = $this->agentName."_dataset";
     $arstable = $this->agentName."_ars";
     /* get proper agent_id */
-    $agentId = GetParm("agent", PARM_INTEGER);
+    // $agentId = GetParm("agent", PARM_INTEGER);
     if (empty($agentId))
```

Comment:
Can you please tell the reason for commenting the $agentId ?

Reply:
Instead of using `$agentId = GetParm("agent", PARM_INTEGER);` we are using this function `$agentId = LatestAgentpk($uploadId, $arstable);` both for copyright `$agentId` here  https://github.com/itssingh/fossology/blob/sarita/newagent/scancode/src/copyright/ui/HistogramBase.php#L240 and scancode `$agentId` here https://github.com/itssingh/fossology/blob/sarita/newagent/scancode/src/copyright/ui/copyright-hist.php#L58 .